networking_nsxv3/api/rpc.py

```diff
@@ -231,69 +170,82 @@ def plugin(self):
         return self._plugin
 
     @log_helpers.log_method_call
-    def get_port_revision_tuples(self, context, host, limit, created_after):
-        return db.get_port_revision_tuples(context, host, limit, created_after)
+    def get_ports_revisions(self, context, host, limit, offset):
+        return db.get_ports_revisions(context, host, limit, offset)
 
     @log_helpers.log_method_call
-    def get_qos_policy_revision_tuples(self, context, limit, created_after):
-        return db.get_qos_policy_revision_tuples(context, limit, created_after)
-
+    def get_qoses_revisions(self, context, host, limit, offset):
+        return db.get_qoses_revisions(context, host, limit, offset)
+    
     @log_helpers.log_method_call
-    @oslo_messaging.expected_exceptions(exceptions.ObjectNotFound)
-    def get_security_group_revision(self, context, security_group_id):
-        return db.get_security_group_revision(context, security_group_id)
+    def get_security_groups_revisions(self, context, host, limit, offset):
+        return db.get_security_groups_revisions(context, host, limit, offset)
 
     @log_helpers.log_method_call
-    def get_security_group_revision_tuples(
-            self, context, limit, created_after):
-        return db.get_security_group_revision_tuples(
-            context, limit, created_after)
-
-    def has_security_group_tag(
-            self, context, security_group_id, tag_name):
-        return db.has_security_group_tag(
-            context, security_group_id, tag_name)
+    def get_security_group(self, context, security_group_id):
+        id_rev = db.get_security_group_revision(context, security_group_id)
+        if id_rev:
+            return {
+                "id": id_rev[0],
+                "revision_number": id_rev[1],
+                "tags": db.get_security_group_tag(context, security_group_id),
+                "ports": db.get_port_id_by_sec_group_id(context, sec_group_id)
```

Comment:
I can't find `sec_group_id` in the body of the function, shouldn't it be `security_group_id``?